tools/src/command/host.rs

```diff
@@ -0,0 +1,162 @@
+use std::{fs, io::Write, path::PathBuf};
+
+use anyhow::Context;
+use clap::Args;
+
+use crate::utils::cargo;
+
+#[derive(Debug, Args)]
+pub struct HostArgs {
+    #[arg(name = "path")]
+    pub path: PathBuf,
+}
+
+pub fn handle_command(args: HostArgs) -> anyhow::Result<()> {
+    let path = args.path;
+
+    setup_crate(path)
+}
+
+fn setup_crate(host_path: PathBuf) -> anyhow::Result<()> {
+    let host_str = host_path
+        .to_str()
+        .context("path is not a valid UTF-8 string")?;
+
+    let guest_path = host_path.join("src").join("guest");
+    let guest_str = guest_path.to_str().unwrap();
+
+    // run cargo to setup project
+    cargo(None, ["new", host_str])?;
+    cargo(
+        Some(&host_path),
+        [
+            "add",
+            "--git",
+            "https://github.com/nexus-xyz/nexus-zkvm.git",
+            "--branch",
+            "simple-sdk",
+            "nexus-sdk",
+        ],
+    )?;
+
+    let mut fp1 = fs::OpenOptions::new()
+        .append(true)
+        .open(host_path.join("Cargo.toml"))?;
+
+    writeln!(
+        fp1,
+        concat!(
+            "\n",
```

Comment:
I had to add
```
[workspace]
members = [
    "src/guest"
]
```
manually before getting `cargo run -r` working.

Reply:
Oh interesting. Maybe cargo version? Will add.